.github/workflows/continuous-delivery.yml

```
@@ -41,11 +41,3 @@ jobs:
           tags: |
             skyrabot/skyra:latest
             ghcr.io/skyra-project/skyra:latest
```

```suggestion
          tags: ghcr.io/skyra-project/skyra:latest
```

If you remove the block below (Dockerhub) then we also shouldn't publish to Dockerhub.